src/main/java/com/greenfoxacademy/opal/kalendaryo/kalendaryo/model/entity/MergedCalendar.java

```diff
@@ -1,38 +1,55 @@
 package com.greenfoxacademy.opal.kalendaryo.kalendaryo.model.entity;
 
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonInclude.Include;
 import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
 import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
 import javax.persistence.OneToMany;
 import java.util.List;
 
 @Entity
 public class MergedCalendar {
 
     @Id
-    private long userId;
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private long id;
     private String userName;
     private String outputAccount;
     private String outputCalendarId;
 
     @OneToMany(mappedBy = "mergedCalendar")
     List<CalendarId> calendarId;
```

Comment:
This variable should be plural, because it is a list, and there will be a lot of it.